content/en/hosting/analytics/setup-docker-compose.md

```diff
@@ -30,36 +30,121 @@ Configure the `COUCHDB_*` environment variables to connect to your CouchDB insta
 The first time you run the commands from any of the sections below it will need to download many Docker images and will take a while. You'll know it's done when you see `#8 DONE 0.0s` and you are returned to the command line. Be patient!
 {{% /alert %}}
 
-### Separate CouchDB instance 
+### Profiles
+
+Running CHT Sync uses docker compose profiles to organize different sets of services. 
+
+The following profiles are available:
+- `local` - for local model development
+- `production` - recommended setup for production
+- `test` - for automated tests
+
+#### Local
 
 This setup involves starting couch2pg, PostgreSQL, pgAdmin and dbt. It assumes you have a CouchDB instance running, and you updated the `.env` CouchDB variables accordingly.
 
 Run the Docker containers locally and wait for every container to be up and running:
 ```sh
-docker compose -f docker-compose.postgres.yml -f docker-compose.yml up -d
+docker compose --profile local up -d
 ```
 
-You can verify this command worked by running `docker ps`. It should show 4 containers running including couch2pg, dbt, PostgreSQL, and pgAdmin.
+You can verify this command worked by running `docker ps`. It should show four containers running including couch2pg, dbt, PostgreSQL, and pgAdmin.
 
-### Separate CouchDB and PostgreSQL instances
+When developing DBT models, it is helpful to test changes locally before committing them to a remote repository.
 
-This local setup involves starting couch2pg and dbt. It assumes that CouchDB and PostgreSQL instances are run separately from the Docker Compose provided with CHT Sync, and the `.env` variables were updated to match those instances details.
+Set the path to the project to the `DBT_LOCAL_PATH` [environment variable]({{< relref "hosting/analytics/environment-variables" >}}) in `.env`.
```

Comment:
moving up as you'll get an error if you call `compose up` before setting this var to a valid value.

```suggestion
```